web/templates/personal_menu.hbs

```diff
@@ -0,0 +1,157 @@
+<div class="dropdown-menu" id="personal-menu-dropdown" data-simplebar>
+    <div class="arrow"></div>
+    <nav class="inner">
+        <header class="user-info-popover-header">
+            <div class="avatar">
+                <div class="avatar-image{{#if user_is_guest}} guest-avatar{{/if}}" style="background-image: url('{{user_avatar}}');"></div>
+
+                {{#if is_active }}
+                <span class="status-circle {{user_circle_class}} user_circle tippy-zulip-tooltip hidden-for-spectators" data-tippy-content="{{user_last_seen_time_status}}"></span>
+                {{/if}}
+
+            </div>
+            <div class="text-area">
+                <p class="full-name">{{user_full_name}}</p>
+                <p class="user-type">{{user_type}}</p>
+            </div>
+        </header>
+        <section class="popover-menu info_popover_actions" data-user-id="{{user_id}}">
+            <ul class="list">
+                {{#if status_content_available}}
+                <li class="text-item">
+                    <span>
+                        {{#if status_emoji_info}}
+                        {{#if status_emoji_info.emoji_alt_code}}
+                        <span class="emoji_alt_code">&nbsp:{{status_emoji_info.emoji_name}}:</span>
+                        {{else if status_emoji_info.url}}
+                        <img src="{{status_emoji_info.url}}" class="emoji status_emoji" />
+                        {{else}}
+                        <span class="emoji status_emoji emoji-{{status_emoji_info.emoji_code}}"></span>
+                        {{/if}}
+                        {{/if}}
+                        <span class="status_text">
+                            {{status_text}}
+                        </span>
+                    </span>
+                    <a href="" class="clear_status" aria-label="{{t 'Clear status'}}" data-tippy-content="{{t 'Clear your status' }}">
+                        <i class="zulip-icon zulip-icon-x-circle"></i>
+                    </a>
+                </li>
+                <li class="link-item hidden-for-spectators">
+                    <a href="" class="update_status_text">
+                        <i class="zulip-icon zulip-icon-smile"></i>
+                        {{#tr}}Edit status{{/tr}}
+                    </a>
+                </li>
```

Comment:
Since we are updating this in the user card migration #25459 as well.
Might want to split this out in another handlebar inorder to not duplicate code.
https://github.com/zulip/zulip/pull/25459/files#diff-d1c1238c4dc94e7f202b56433ac6b92f8a3f4b7331a54283ac4ae26cb21645d8R77-R98

btw missing semi-colon in `&nbsp;`

Reply:
and `hidden-for-spectator` class is not required right ?

Reply:
> Since we are updating this in the user card migration #25459 as well. Might want to split this out in another handlebar inorder to not duplicate code. https://github.com/zulip/zulip/pull/25459/files#diff-d1c1238c4dc94e7f202b56433ac6b92f8a3f4b7331a54283ac4ae26cb21645d8R77-R98
> 
> btw missing semi-colon in `&nbsp;`

I will fix the semi-colon in the updated PR. 

For the split-out part, you suggested I am thinking of doing it in user card redesign PR as not only this but almost all of the design is the same in the profile menu and user card, you can check this is #22802. Thoughts on this plan?

Reply:
Fixed both Issues and also fixed the dropdown disappearing on resize in updated PR.